README_ENGLISH.md

```diff
@@ -0,0 +1,58 @@
+# Project Buy It
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=salomaoluiz_BuyIt&metric=alert_status)](https://sonarcloud.io/dashboard?id=salomaoluiz_BuyIt)
+
+This project has be created as a study of case several at development points. If you want use this project to study or commercially, feel free, as long as you follow the [LICENSE](https://github.com/salomaoluiz/BuyIt/blob/main/LICENSE) rules. Most of this project has the descriptions in portuguese, because this way is more easily to a brazilian starter developer understant without language trouble, just the code is fully in english.
+
+## Firebase
+For this project is used the Firebase as a BAS (Backend as a Service), soo you need realize some settings in your project before start.
+
+First, you need create a project in Firebase (Two projects if you wanna separe the development environment of production environment), configure the project to Android and add the **google-services.json** in folders *./android/app/src/development* and *./android/app/src/production*.
+
+Realized this initial settings, you need adjust the ADMOB for the ads, to configure this is required create 3 files, with each one theys with the following informations:
```

Comment:
```suggestion
After that, you need to adjust the *ADMOB* for the ads, to configure this, you need to create 3 files, each file with the following information:
```

Reply:
@salomaoluiz There is no information, where these 3 files need to be created, I mean in which folder these files should be?

Reply:
These files need to be created in root folder, together with *package.json*.

Reply:
Look if this way is better.

Reply:
I have updated my edit, mentioning the three files need to be created in root folder

Reply:
Nice, I will change, thanks.